Modelica/Electrical/Analog/UsersGuide/ReleaseNotes.mo

```diff
@@ -3,7 +3,7 @@ class ReleaseNotes "Release Notes"
   extends Modelica.Icons.ReleaseNotes;
   annotation (preferredView="info",Documentation(info="<html>
 
-<h5>Version 4.0.0, 2020-06-04</h5>
+<h5>Version 4.1.0, 2024-03-15</h5>
```

Comment:
I think this is wrong. It is a Release Notes item for 4.0.0, so the version and date shouldn't change?

Reply:
Agreed.
It also indicates that the idea of adding ReleaseNotes to sub-libraries in #2990 might not be a good idea, and to me it would be preferable if we had the following policy:
- Don't _add_ ReleaseNotes to sub-libraries.
- Keep but do not update ReleaseNotes. Ideally the last entry in a sub-package should be the statement that it was added to MSL.

Which means that it was an error to add this ReleaseNotes class.
I know that there are other opinions, but to me it seems to take too much effort to update the sub-package release notes, and thus it isn't done consistently. That means that it is not useful for figuring out what was changed.